redis/src/aio/tokio.rs

```diff
@@ -100,6 +102,7 @@ impl RedisRuntime for Tokio {
         hostname: &str,
         socket_addr: SocketAddr,
         insecure: bool,
+        ca_cert: Option<&Certificate>,
```

Comment:
I think we should take an `Option<Certificate>` here, so that cloning is ultimately left to the caller.

Reply:
Done